README.md

```diff
@@ -2,22 +2,14 @@
 
 Core ocean sonar data visualization functionalities based on the HoloViz suite of tools.
 
+## What are ocean sonar systems?
+
 Ocean sonar systems, such as echosounders, are the [workhorse to study life in the ocean](https://storymaps.arcgis.com/stories/e245977def474bdba60952f30576908f). They provide continuous observations of fish and zooplankton by transmitting sounds and analyzing the echoes bounced off these animals, just like how medical ultrasound images the interior of human body. In recent years these systems are widely deployed on ships, autonomous vehicles, or moorings, bringing in a lot of data that allow scientists to study the change of the marine ecosystem.
 
 ## Google Summer of Code
 
-Hey there! Welcome 😃.
-
-Open Source Ocean Acoustics started back in 2018 from Oceanhackweek. It is meant as a home for open source tools and resources in ocean acoustics. We are a diverse group of researchers whose work centers around extracting knowledge from large volumes of ocean acoustic data, which contain rich information about animals ranging from zooplankton, fish, to marine mammals. Integrating physics-based models and data-driven methods, our current work focuses on mining water column sonar data and spans a broad spectrum from developing computational methods, building open source software and cloud applications, to joint analysis of acoustic observations and ocean environmental variables.
-
-Echoshader aims to recruit talented [Google Summer of Code (GSoC)](https://summerofcode.withgoogle.com/) participants to help us enhance the capability to interactively visualize large volumes of cloud-hosted ocean sonar data to accelerate the the data exploration and discovery process. The project will go hand-in-hand with ongoing development of the [echopype](https://github.com/OSOceanAcoustics/echopype) package that handles the standardization, pre-processing, and organization of these data.
-
-In the [Issues](https://github.com/OSOceanAcoustics/echoshader/issues) section of this repository, we list project ideas from mentors. We encourage you as a GSoc participant to propose your own original project ideas as new issues.
-
-After you [sign up as a GSoC participant](https://summerofcode.withgoogle.com/get-started/) Please submit your proposals based on the [Echoshader GSoC Proposal](proposal-template.md) template.
-
-### Questions?
+Hey there! Welcome! 😃
 
-Please open an issue within this repo for any questions that you may have.
+This project aims to enhance the capability to interactively visualize large volumes of cloud-hosted ocean sonar data to accelerate the the data exploration and discovery process. The project will go hand-in-hand with ongoing development of the echopype package that handles the standardization, pre-processing, and organization of these data.
```

Comment:
cloud-hosted should change to cloud-based since now we are dealing with the cloud-native format of zarr not just the hosted raw files

Reply:
- "the the" > "the"
- echopype > [echopype](https://echopype.readthedocs.io/en/stable/)